pkg/driver/mocks/mock_mounter.go

```diff
@@ -35,6 +40,20 @@ func (m *MockMounter) EXPECT() *MockMounterMockRecorder {
 	return m.recorder
 }
 
+// CanSafelySkipMountPointCheck mocks base method.
+func (m *MockMounter) CanSafelySkipMountPointCheck() bool {
```

Comment:
The mockgen seems to have exported this method, although this method is not seen to be used throughout the repository.